### PR TITLE
Update de.po

### PR DESCRIPTION
**Pull request by 768kb**
Some corrections and suggestions

### CODE REVIEW

File: de.po [hunk 1]
```diff
@@ -2077,7 +2077,7 @@ msgstr "Speicherpools"
 
 #: src/components/create-vm-dialog/createVmDialog.jsx:173
 msgid "Storage size must not be 0"
-msgstr ""
+msgstr "Speichergrösse darf nicht 0 sein"
```
Comment (martinpitt) [inline de.po:2080]: It's spelled "Größe" (also in the strings below).

File: de.po [hunk 2]
```diff
@@ -2086,15 +2086,15 @@ msgstr ""
 #: src/components/vm/disks/diskAdd.jsx:334
 #: src/components/storagePools/storageVolumeCreate.jsx:64
 msgid "Storage volume size must not exceed the storage pool's capacity ($0 $1)"
-msgstr ""
+msgstr "Die Grösse des Datenträgers darf nicht die Grösse der Storage Pool Kapazität $0 $1 übersteigen"
```
Comment (martinpitt) [inline de.po:2089]: Please use hyphens, as we do in German: Storage-Pool-Kapazität (plus Größe)

File: de.po [hunk 3]
```diff
@@ -2176,16 +2176,16 @@ msgstr "Der Pool ist leer"
 
 #: src/components/create-vm-dialog/createVmDialog.jsx:420
 msgid "The selected operating system does not support unattended installation"
-msgstr ""
+msgstr "Das ausgewählte Betriebssystem unterstützt keine automatische Installation"
 
 #: src/components/create-vm-dialog/createVmDialog.jsx:597
 msgid "The selected operating system has minimum memory requirement of $0 $1"
-msgstr ""
+msgstr "Das ausgewählte Betriebssystem hat eine minimum Arbeitsspeicher-Anforderung von $0 $1"
```
Comment (martinpitt) [inline de.po:2183]: "minimale"

File: de.po [hunk 4]
```diff
@@ -2198,15 +2198,15 @@ msgstr ""
 
 #: src/components/create-vm-dialog/createVmDialog.jsx:687
 msgid "This volume is already used by another VM."
-msgstr ""
+msgstr "Dieser Datenträger ist bereits in Verwendung einer anderen VM."
```
Comment (martinpitt) [inline de.po:2201]: This sounds a bit weird IMHO. Maybe "Dieser Datenträger wird bereits von einer anderen VM verwendet."?

File: de.po [hunk 5]
```diff
@@ -2198,15 +2198,15 @@ msgstr ""
...
 
 #: src/components/vm/disks/diskAdd.jsx:208
 msgid "This volume is already used by: "
-msgstr ""
+msgstr "Dieser Datenträger ist bereits in Verwendung von:"
 
 #: src/components/vm/overview/vcpuModal.jsx:233
 msgid "Threads per core"
-msgstr "Fäden pro Kern"
+msgstr "Threads pro Core"
```
Comment (martinpitt) [inline de.po:2209]: Thanks, "Fäden" is ridiculous, but "Kern" is fine IMHO.

File: de.po [hunk 6]
```diff
@@ -2279,16 +2279,16 @@ msgstr "Unbekannte Firmware"
 
 #: src/components/vm/nics/vmNicsCard.jsx:303
 msgid "Unplug"
-msgstr "Ziehen Sie den Stecker heraus"
+msgstr "trennen"
```
Comment (martinpitt) [inline de.po:2282]: Please capitalize, "Trennen".

File: de.po [hunk 7]
```diff
@@ -2279,16 +2279,16 @@ msgstr "Unbekannte Firmware"
...
 
 #: src/components/create-vm-dialog/createVmDialog.jsx:635
 msgid "Up to $0 $1 available on the default location"
-msgstr ""
+msgstr "Bis zu $0 $1 verfügbar in der Standard Lokation"
```
Comment (martinpitt) [inline de.po:2286]: "Lokation" sounds weird, and hyphens. "im Standard-Ort"? or "Standard-Pool"?

File: de.po [hunk 8]
```diff
@@ -2315,19 +2315,19 @@ msgstr "Benutzt von"
 #: src/components/create-vm-dialog/createVmDialog.jsx:539
 #, fuzzy
```
Comment (martinpitt) [inline de.po:2316]: You need to remove fuzzy flag after updating strings.

File: de.po [hunk 9]
```diff
@@ -2315,19 +2315,19 @@ msgstr "Benutzt von"
...
 msgid "User login"
-msgstr "Letzte Anmeldung"
+msgstr "Benutzer Anmeldung"
```
Comment (martinpitt) [inline de.po:2318]: I'd say, "Benutzername" is clearer here.

File: de.po [hunk 10]
```diff
@@ -2315,19 +2315,19 @@ msgstr "Benutzt von"
...
 
 #: src/components/create-vm-dialog/createVmDialog.jsx:187
 msgid "User login must not be empty when user password is set"
-msgstr ""
+msgstr "Benutzer Login darf nicht leer sein wenn Passwort gesetzt ist"
```
Comment (martinpitt) [inline de.po:2322]: Please use a consistent wording here, same as "User login" above. "Benutzername" is our standard term in cockpit.

File: de.po [hunk 11]
```diff
@@ -2315,19 +2315,19 @@ msgstr "Benutzt von"
...
 
 #: src/components/create-vm-dialog/createVmDialog.jsx:546
 msgid "User password"
-msgstr "Benutzerpasswort"
+msgstr "Benutzer Passwort"
```
Comment (martinpitt) [inline de.po:2326]: "Benutzerpasswort" or "Benutzer-Passwort"

File: de.po [hunk 12]
```diff
@@ -2451,19 +2451,19 @@ msgstr "Der Virtualisierungsdienst (libvirt) ist nicht aktiv"
 #: src/components/vm/overview/bootOrder.jsx:95
 #: src/components/create-vm-dialog/createVmDialog.jsx:685
 msgid "Volume"
-msgstr "Lautstärke"
+msgstr "Datenträger"
 
 #: src/components/storagePools/storageVolumeCreate.jsx:83
 msgid "Volume failed to be created"
-msgstr ""
+msgstr "Fehler beim erstellen des Datenträgers"
```
Comment (martinpitt) [inline de.po:2458]: "E"rstellen

File: de.po [hunk 13]
```diff
@@ -2451,19 +2451,19 @@ msgstr "Der Virtualisierungsdienst (libvirt) ist nicht aktiv"
...
 
 #: src/components/storagePools/createStoragePoolDialog.jsx:168
 msgid "Volume group name"
-msgstr ""
+msgstr "Datenträger Gruppenname"
```
Comment (martinpitt) [inline de.po:2462]: IMHO that's too confusing -- "volume group" is an LVM term. Cockpit's German translations use "Datenträgerverbund", but just keeping the term may be less confusing for admins. "Name der Volume-Group"?

File: de.po [hunk 14]
```diff
@@ -2611,12 +2611,12 @@ msgstr "Nein"
...
 #: src/components/vm/snapshots/vmSnapshotsCard.jsx:142
 msgid "no state saved"
-msgstr ""
+msgstr "kein Zustand gesichert"
 
 #: src/lib/cockpit-components-password.jsx:85
 #, fuzzy
 msgid "password quality"
-msgstr "Passwort"
+msgstr "Passwort Qualität"
```
Comment (martinpitt) [inline de.po:2619]: Hyphen or single word again.

**Review by martinpitt — CHANGES_REQUESTED** [review]
Danke! We normally don't edit the .po files directly, but edit translations on https://translate.fedoraproject.org/projects/cockpit-machines/main/de/

Please let me know if you still prefer doing this here, then I'm happy to copy them to weblate once you are done.

Some suggestions, though.

**Comment by marusak** [comment]
This is out of date. I am closing this mainly as it should be submitted in Weblate. @768kb please let me know if you need help with Weblate.

**Pull request closed without merging**